x-pack/plugins/ml/public/application/jobs/jobs_list/components/job_details/job_messages_pane.tsx

```diff
@@ -46,18 +61,89 @@ export const JobMessagesPane: FC<JobMessagesPaneProps> = React.memo(
 
     const refreshMessage = useCallback(fetchMessages, [jobId]);
 
+    // Clear messages for last 24hrs and refresh jobs list
+    const clearMessages = useCallback(async () => {
+      setIsClearing(true);
+      try {
+        await clearJobAuditMessages(jobId);
+        setIsClearing(false);
+        // wait half a second before refreshing the jobs list
+        // setTimeout(refreshJobList, 500);
+        if (typeof refreshJobList === 'function') {
+          refreshJobList();
+        }
+      } catch (e) {
+        setIsClearing(false);
+        toastNotificationService.displayErrorToast(
+          e,
+          i18n.translate('xpack.ml.jobMessages.clearJobAuditMessagesErrorTitle', {
+            defaultMessage: 'Error clearning job message warnings and errors',
```

Comment:
Typo - `clearing`

Reply:
Good catch - updated in 4103bbf25334252c315e0ee2c5090711e9325fd2